docs/enter-metagame/how-to-become-a-player.md

```diff
@@ -20,7 +20,7 @@ Ain't no freeloading in MetaGame. Luckily, there are countless ways of contribut
 -   Least you can do is report on your progress, whether its something you're learning or the project you're working on. Just _[#](https://discord.gg/dU2pnEp)_[🌞](https://discord.gg/dU2pnEp)_[-check-in](https://discord.gg/dU2pnEp)._
     
 
-To get inspired on how you could contribute: read through [the roles](https://wiki.metagame.wtf/docs/enter-metagame/roles-in-metagame) or check out the _[ongoing raids](https://wiki.metagame.wtf/docs/enter-metagame/navigation-board)_. Find the role that fits you best, a raid you are most drawn to, or just do whatever you think is valuable & report in _[#](https://discord.gg/JAFX3Ry)_[😊](https://discord.gg/JAFX3Ry)_[-I-did-this](https://discord.gg/JAFX3Ry)_.
+To get inspired on how you could contribute: read through [the roles](https://wiki.metagame.wtf/docs/enter-metagame/roles-in-metagame) or check out the _[ongoing raids](https://wiki.metagame.wtf/docs/enter-metagame/navigation-board)_. Find the role that fits you best, a raid you are most drawn to, or just do whatever you think is valuable & report in _[#](https://discord.gg/JAFX3Ry)_[😊](https://discord.gg/JAFX3Ry)_[-did-a-thing](https://discord.gg/JAFX3Ry)_.
```

Comment:
@SLRead thanks for sorting this but we have a conflict here where you've rightly fixed the did-a-thing channel name but it's already been fixed and merged in a PR from someone else. Sorry about that.

Reply:
No worries!  I'll remove the conflict and get this sorted!  Thanks for reviewing!